cli/cmd/install/install.go

```diff
@@ -232,23 +296,87 @@ timescaledb-single:
 		fmt.Println("skipping the wait for pods to come to a running state because --skip-wait is enabled.")
 	}
 
-	fmt.Println("The Observability Stack has been installed successfully")
-	fmt.Println(release.Info.Notes)
+	// create the default otelcol CR as operator is up & running by now....
+	if c.enableOtel {
+		otelOperatorNamespace := "opentelemetry-operator-system"
+		// check the status of otel operator as CR creation needs webhooks
+		// validation from operator
+		otelOperatorPod, err := k8sClient.KubeGetPodName(otelOperatorNamespace, map[string]string{"control-plane": "controller-manager"})
+		if err != nil {
+			return fmt.Errorf("failed to find otel operator: %v", err)
+		}
+		err = k8sClient.KubeWaitOnPod(otelOperatorNamespace, otelOperatorPod)
+		if err != nil {
+			return err
+		}
+
+		err = k8sClient.CreateDefaultOtelCol(cmd.HelmReleaseName, cmd.Namespace, c.OtelColConfig)
+		if err != nil {
+			return err
+		}
+		fmt.Println("created the opentelemetry-collector custom resource...")
+	}
+
+	if release.Info != nil {
+		fmt.Println(release.Info.Notes)
+		fmt.Println("The Observability Stack has been installed successfully")
+	} else {
+		fmt.Println("failed to install tobs completely, release notes generation failed...")
+	}
+
 	return nil
 }
 
-func appendDBURIValues(dbURI, name string, helmValues string) string {
+func appendDBURIValues(dbURI, helmValues string) string {
 	helmValues = helmValues + fmt.Sprintf(`
 timescaledb-single:
   enabled: false
 timescaledbExternal:
   enabled: true
-  db_uri: %s
-promscale:
+  db_uri: %s`, dbURI)
+	return helmValues
+}
+
+func enableOtelInValues(helmValues string) string {
+	helmValues = helmValues + `
+opentelemetryOperator:
+  enabled: true
+`
+	return helmValues
+}
+
+func appendPromscaleValues(enableOtel, promHA bool, dbURI string) string {
```

Comment:
As multiple times we want to update `promscale` field values i.e.

```
promscale:
  image: xyz
```

```
promscale:
  replicaCOunt: 3
```

This way of configuring using CLI is overriding previous `promscale:` map values. So this `func` helps to append values within `promscale`.